File: src/library/Shift.Sdk/Contract/Evaluation/Workshops/Models/WorkshopQuestionOption.cs
namespace Shift.Contract
{
    public class WorkshopQuestionOption
    {
        public class OptionColumn
        {
            public string TextMarkdown { get; set; }
            public string TextHtml { get; set; }            
        }

        public int Number { get; set; }
        public string Letter { get; set; }
        public string TitleMarkdown { get; set; }
        public string TitleHtml { get; set; }
        public int Points { get; set; }
        public bool? IsTrue { get; set; }
        public OptionColumn[] Columns { get; set; }
    }
}

File: src/ui/Shift.UI/src/contexts/workshop/models/WorkshopQuestionOption.ts
export interface WorkshopQuestionOption {
    number: number;
    letter: string;
    titleMarkdown: string | null;
    titleHtml: string | null;
    points: number;
    isTrue: boolean | null;
    columns: {
        textMarkdown: string | null;
        textHtml: string | null;
    }[] | null,
}